exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/parser/UnsupportedOperatorsVisitor.java

```diff
@@ -305,6 +313,105 @@ public SqlNode visit(SqlCall sqlCall) {
     return sqlCall.getOperator().acceptCall(this, sqlCall);
   }
 
+  private void checkRollupCubeGrpSets(SqlSelect sqlSelect) {
+    final ExprFinder rollupCubeGrpSetsFinder = new ExprFinder(RollupCubeGrpSets);
+    sqlSelect.accept(rollupCubeGrpSetsFinder);
+    if (rollupCubeGrpSetsFinder.find()) {
+      unsupportedOperatorCollector.setException(SqlUnsupportedException.ExceptionType.FUNCTION,
```

Comment:
Good point. I'll create an enhancement JIRA and point to the enhancement JIRA in the error msg.